rclcpp/include/rclcpp/node.hpp

```diff
@@ -580,6 +580,22 @@ class Node : public std::enable_shared_from_this<Node>
     > & parameters,
     bool ignore_override = false);
 
+  /// Declare and initialize several parameters with the same namespace.
+  /**
+   * This version will take a vector where the value is a pair, with the
+   * rclcpp::Parameter object as the first item and a parameter descriptor
+   * as the second.
+   *
```

Comment:
The interface description isn't enough.    
Such as the calling  `add_on_set_parameters_callback` and `add_post_set_parameters_callback`  are different from original interface declare_parameters.  

Please refer to the description of interface set_parameters_atomically.